includes/apple-exporter/builders/class-components.php

```diff
@@ -30,7 +30,67 @@ protected function build() {
 		// such as if a thumbnail was used from the body.
 		$components = array_merge( $this->meta_components(), $components );
 
-		return $this->group_body_components( $components );
+		$grouped_components = $this->group_body_components( $components );
+		$grouped_components = $this->componentize( $grouped_components );
+
+		return $grouped_components;
+	}
+
+	/**
+	 * Give an array of components in array format, transform
+	 * all that are needed into components
+	 *
+	 * @param array $components
+	 * @return array
+	 * @access private
+	 */
+	private function componentize( $components ) {
```

Comment:
I think the `componentize` function isn't necessary and I'll add some notes as to why.